Mathlib/GroupTheory/PresentedGroup.lean

```diff
@@ -94,6 +94,14 @@ theorem ext {φ ψ : PresentedGroup rels →* G} (hx : ∀ (x : α), φ (.of x)
   ext
   apply hx
 
+/-- Presented groups of isomorphic types are isomorphic. -/
+def equivPresentedGroup {β : Type*} (e : α ≃ β) :
+    PresentedGroup rels ≃* PresentedGroup ((FreeGroup.freeGroupCongr e) '' rels) :=
+  QuotientGroup.congr (Subgroup.normalClosure rels)
+    (Subgroup.normalClosure ((FreeGroup.freeGroupCongr e)'' rels)) (FreeGroup.freeGroupCongr e)
+    (Subgroup.map_normalClosure rels (FreeGroup.freeGroupCongr e).toMonoidHom
+      (FreeGroup.freeGroupCongr e).surjective)
```

Comment:
I think you want `rels` to be an explicit argument to `equivPresentedGroup`, right? That way you can write `equivPresentedGroup rels e` and specify `rels`.

Definitions should generally come with some minimal API that characterizes them. The main one I can think of is `equivPresentedGroup_apply_of` (if I got the name right). That would be something like `equivPresentedGroup rels e (PresentedGroup.of x) = PresentedGroup.of (e x)`. You could also throw in `equivPresentedGroup_symm_apply_of` for `(equivPresentedGroup rels e).symm (PresentedGroup.of x) = PresentedGroup.of (e.symm x)`, and you probably can deduce it from the previous one.